kpi/serializers/v2/asset_permission_assignment.py

```diff
@@ -318,11 +309,13 @@ class AssetBulkInsertPermissionSerializer(serializers.Serializer):
     Warning: If less queries are sent to DB, it consumes more CPU and memory.
     The bigger the assignments are, the bigger the resources footprint will be.
     """
+
     assignments = serializers.ListField(child=PermissionAssignmentSerializer())
 
     @dataclass(frozen=True)
     class PermissionAssignment:
-        """ A more-explicit alternative to a simple tuple """
+        """A more-explicit alternative to a simple tuple"""
```

Comment:
Your linter really don't like when the code breaths a little :-P 
I would leave the extra space alone or even better I would put the docstrings on separated  lines.

```python
"""
A more-explicit alternative to a simple tuple
"""
```

I prefer this way even if the sentence is short. Moreover if there is a diff at a later time, it does not involve the triple quotes.